changelog/james-prysm_safe-validator-shutdown.md

```diff
@@ -0,0 +1,11 @@
+## Added
+
+- Added `max-health-checks` flag that sets the maximum times the validator tries to check the health of the beacon node before timing out. 0 or a negative number is indefinite.
+
+## Fixed
+
+- Validator client shuts down cleanly on error instead of fatal error. 
+
+## Changed
+
+- If beacon node disconnects can now time out on retries 
```

Comment:
Incomplete sentence

Reply:
i should remove this since we changed the default to 0